src/Database/Adapter.php

```diff
@@ -490,4 +491,18 @@ public function filter(string $value): string
 
         return $value;
     }
+
+    /**
+     * Increase and Decrease Attribute Value
+     *
+     * @param string $collection
+     * @param string $id
+     * @param string $attribute
+     * @param int|float $value
+     * @param int|null $min
+     * @param int|null $max
+     * @return bool
+     * @throws Exception
+     */
+    abstract function increaseDocumentAttribute(string $collection, string $id, string $attribute, int|float $value, int $min = null, int $max = null):bool;
```

Comment:
```suggestion
    abstract function increaseDocumentAttribute(string $collection, string $id, string $attribute, int|float $value, ?int $min = null, ?int $max = null):bool;
```

If min and max are nullable shouldn't they be like this ?